workflow/scripts/enrichment_pathway_scatter.py

```
@@ -0,0 +1,183 @@
+import pandas as pd
+import altair as alt
+import sys
+
+sys.stderr = open(snakemake.log[0], "w")
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)